m68k-amigaos.cmake

```
@@ -32,21 +31,18 @@ set(M68K_FPU "soft" CACHE STRING "FPU type")
 set_property(CACHE M68K_FPU PROPERTY STRINGS ${M68K_FPU_TYPES})
 
 # CRT
-set(MCRT_TYPES "nix20" "nix13" "clib2" "ixemul" "newlib")
-set(MCRT "nix20" CACHE STRING "Target std lib")
-set_property(CACHE MCRT PROPERTY STRINGS ${MCRT_TYPES})
+set(M68K_CRT_TYPES "nix20" "nix13" "clib2" "ixemul" "newlib")
+set(M68K_CRT "nix20" CACHE STRING "Target std lib")
+set_property(CACHE M68K_CRT PROPERTY STRINGS ${M68K_CRT_TYPES})
 
 # Extra flags
 set(TOOLCHAIN_CFLAGS "${M68K_CFLAGS}" CACHE STRING "CFLAGS")
 set(TOOLCHAIN_CXXFLAGS "${M68K_CXXFLAGS}" CACHE STRING "CXXFLAGS")
 set(TOOLCHAIN_LDFLAGS "${M68K_LDFLAGS}" CACHE STRING "LDFLAGS")
 set(TOOLCHAIN_COMMON "${M68K_COMMON}" CACHE STRING "Common FLAGS")
 
-if(NOT M68K_TOOLCHAIN_PATH)
-	set(TOOLCHAIN_PATH /opt/${TOOLCHAIN_PREFIX})
-else()
-	set(TOOLCHAIN_PATH ${M68K_TOOLCHAIN_PATH})
-endif()
+set(TOOLCHAIN_PATH_DEFAULT /opt/amiga)
```

```suggestion
set(TOOLCHAIN_PATH_DEFAULT /opt/${TOOLCHAIN_PREFIX})
```
I missed this one, this is going to break things, there is a reason why set(TOOLCHAIN_PATH /opt/${TOOLCHAIN_PREFIX}) was there.

TOOLCHAIN_PREFIX is m68k-amigaos, which means it looks for m68k-amigaos-gcc. The reason why you can set the toolchain prefix, is because if you run CMake on the target machine, it would just be called gcc.

All toolchains look for /opt/${TOOLCHAIN_PREFIX} so, /opt/ppc-morphos, /opt/m68k-amigaos etc.